### ISSUE TITLE
linter generates warnings during build due to Json function names

### ISSUE THREAD

**Issue by adshmh** [kubernetes/helm#3747]
linter generates several warnings during build:

```
pkg/chartutil/files.go:214:6:warning: func ToJson should be ToJSON (golint)
pkg/chartutil/files.go:229:6:warning: func FromJson should be FromJSON (golint)
```

**Comment by jascott1** [comment]
I labeled this a bug because it either needs to be fixed or silenced.

**Comment by bacongobbler** [comment]
These functions are publicly facing APIs so we cannot change them, but gometalinter does allow [comment directives](https://github.com/alecthomas/gometalinter#comment-directives) to disable these warnings.

So, we need to

1. add `// nolint` to those two lines
2. add another TODO comment to change the function signatures in Helm 3

This should be a super simple PR if anyone wants to make a contribution!

**Comment by cameronconradt** [comment]
I'd like to help out with this, so I just add the comments like this?

`func FromJson(str string) map[string]interface{} { //nolint`

Also where would I add the TODO comment? Just above those functions?

**Comment by bacongobbler** [comment]
yep! pretty much. :)

**Comment by bacongobbler** [comment]
Fixed in #3950
